app/models/sdg/target.rb

```diff
@@ -0,0 +1,36 @@
+class SDG::Target < ApplicationRecord
+  include Comparable
+
+  validates :code, presence: true, uniqueness: true
+  validates :goal, presence: true
+
+  belongs_to :goal
+
+  def title
+    I18n.t("sdg.goals.goal_#{goal.code}.targets.target_#{code_key}.title")
+  end
+
+  def <=>(target)
```

Comment:
I forgot to add a line here:

```
    return unless target.class == self.class
```

And then we should add a test where we compare it with another object.

This bug was found when trying to use `match_array` in a spec when the array contained both targets and goals.